setup.py

```diff
@@ -14,7 +14,9 @@
     "Topic :: Scientific/Engineering :: Bio-Informatics",
 ]
 
-install_requires = [line.rstrip() for line in open(os.path.join(os.path.dirname(__file__), "REQUIREMENTS.txt"))]
+install_requires = [
+    line.rstrip() for line in open(os.path.join(os.path.dirname(__file__), "REQUIREMENTS-DEV.txt"))
```

Comment:
If we do this, what's the point of `REQUIREMENTS.txt` and `REQUIREMENTS-DEV.txt`?  Why not just trash the former?

Reply:
Agree, it loses its importance. How do you feel about this change?

Reply:
I still want starfish-lite and starfish-ezmode. :)

Reply:
Ok. Practically, how do we get the docs building again in the short term? Do you want me to rename `REQUIREMENTS-DEV.txt` and remove `REQUIREMENTS.txt`?